.github/workflows/sync-from-upstream.yml

```diff
@@ -15,10 +15,25 @@ jobs:
       github.repository != 'panther-labs/panther-analysis'
     runs-on: ubuntu-latest
     steps:
+      # Set token based on what's available
+      - name: Set token
+        id: set_token
+        run: |
+          # Check if PANTHER_SYNC_UPSTREAM secret is available by trying to substitute it
+          MASKED_PAT="${{ secrets.PANTHER_SYNC_UPSTREAM }}"
+          if [ -n "$MASKED_PAT" ]; then
```

Comment:
Does this prevent the string from being in plaintext or something? 😊

Reply:
It is, since it's being referenced as `${{ secrets.SECRET_NAME }}`